### PR TITLE
Update indicators paper; add perspective piece

### LINKED ISSUE #464: Correct hero image for indicators paper
When we go to update the carousel, I'm not sure where to find the large hero images for any given publication.  In content/about/publications/images/, we seem to have images of all different sizes, but not necessarily the big wide format that we need for the hero image. 

I'm looking in particular for a big wide version of indicators.jpg.  Thanks!

### PR DESCRIPTION
**Pull request by christymelucci**
Add ‘indicators-hero.jpg’ (1440x500)

Fixes #464

### CODE REVIEW

**Comment by christymelucci** [comment]
Added hero image for indicators paper, then pointed hero to image

**Comment by christymelucci** [comment]
Updated the indicators paper author list to include the Google Fellows

File: content/about/publications/index.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ papers:
     year: 2021
   - title: "An Open Repository of Real-Time COVID-19 Indicators"
     image: indicators.jpg
-    authors: Reinhart, Brooks, Jahja, Rumack, Tang, Al Saeed, Arnold, Basu, Bien, Cabrera, Chin, Chua, Clark, DeFries, Forlizzi, Gratzl, Green, Haff, Han, Hu,  Hyun, Joshi, Kim, Kuznetsov, La Motte-Kerr, Jin Lee, Lee, Lipton, Liu, Mackey, Mazaitis, McDonald, Narasimhan, Oliveira, Patil, Perer, Politsch, Rajanala, Rucker, Shah, Shankar, Sharpnack, Shemetov, Simon, Srivastava, Tan, Tibshirani, Tuzhilina, Van Nortwick, Ventura, Wasserman, Weiss, Williams, Rosenfeld, Tibshirani
+    authors: Reinhart, Brooks, Jahja, Rumack, Tang, Agrawal, Al Saeed, Arnold, Basu, Bien, Cabrera, Chin, Chua, Clark, Colquhoun, DeFries, Farrow, Forlizzi, Grabman, Gratzl, Green, Haff, Han, Harwood, Hu, Hyde, Hyun, Joshi, Kim, Kuznetsov, La Motte-Kerr, YJ Lee, K Lee, Lipton, Liu, Mackey, Mazaitis, McDonald, McGuinness, Narasimhan, O’Brien, Oliveira, Patil, Perer, Politsch, Rajanala, Rucker, Scott,  Shah, Shankar, Sharpnack, Shemetov, Simon, Smith,  Srivastava,  Tan, R Tibshirani, Elena Tuzhilina, Van Nortwick, Ventura, Wasserman, Weaver, Weiss, Whitman, Williams, Rosenfeld, RJ Tibshirani
```
Comment (krivard) [inline content/about/publications/index.md:18]: Elena Tuzhilina is the only entry with a full first name, is that intentional?

**Comment by krivard** [comment]
Wow that's still applying really terrible jpeg compression, trying a different crop

**Comment by krivard** [comment]
okay that's less bad

**Review by krivard — APPROVED** [review]

**Comment by christymelucci** [comment]
Adding new perspective image related to "The Road Ahead" close in the perspective piece.
Check: Does this image take the balance over the top in the nature category for the research page? If leaning toward 'yes', is it reasonable to keep it (for research page + add hero version) for now so the piece posts, then update later if a more compelling image can be found?

**Review by krivard — APPROVED** [review]
Nature images are compelling; I don't think we need to worry about overdoing them

**Pull request merged**